docs/v3.x/admin-panel/deploy.md

```diff
@@ -50,5 +50,5 @@ After running `yarn build` with this configuration, the folder `build` will be c
 The administration URL will then be `http://yourfrontend.com` and every request from the panel will hit the backend at `http://yourbackend.com`.
 
 ::: tip NOTE
-If you add a path to the `url` option, it won't prefix your app. To do so, you need to also use a proxy server like Nginx. More [here](../getting-started/deployment#optional-software-guides).
+If you add a path to the `url` option, it won't prefix your app. To do so, you need to also use a proxy server like Nginx. More [here](../getting-started/deployment.html#optional-software-guides).
```

Comment:
can you please change this to a `.md` instead of `.html` Vuepress will create the correct link for us during the build.

Reply:
@derrickmehaffy ohhh sorry on that. I have never worked with Vuepress, my bad. Changed as requested